repository/src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication-context.xml

```diff
@@ -155,6 +155,9 @@
       <property name="adminConsoleRedirectPath">
          <value>${identity-service.admin-console.redirect-path}</value>
       </property>
+      <property name="signatureAlgorithms">
+         <value>${identity-service.signatureAlgorithms:RS256,PS256}</value>

```

Comment:
```suggestion
         <value>${identity-service.signature-algorithms:RS256,PS256}</value>
```

Given the style of the other properties in this class I think we should try to keep it consistent, unless there's a good reason why we didn't in the first place

Reply:
Changed.